Samples/iOS-Swift6/Info.plist

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`m`: Just so we are aligned, if we are not including the Xcode project in the repository anymore, because it can be generated, shouldn't we also remove Info.plist because it is also generated?

Reply:
for most of these, i turned off plist generation. i don't feel strongly one way or the other, as we can have a shared xcconfig with plist settings, or a shared plist file, in the future. let's make the decision in a later refactor PR.